mlflow/resource_experiment_test.go

```diff
@@ -0,0 +1,115 @@
+package mlflow
+
+import (
+	"testing"
+
+	"github.com/databrickslabs/terraform-provider-databricks/qa"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMLFlowExperimentCreate(t *testing.T) {
+	d, err := qa.ResourceFixture{
+		Fixtures: []qa.HTTPFixture{
+			{
+				Method:   "POST",
+				Resource: "/api/2.0/mlflow/experiments/create",
+				ExpectedRequest: MLFLowExperimentAPI{
+					Name: "xyz",
+					Tags: []Tag{
+						{"key1", "value1"},
+						{"key2", "value2"},
+					},
+				},
+				Response: MLFLowExperimentAPI{
```

Comment:
The MLFlowExperimentAPI can be simplified to a single variable, that gets used here and in the next response.